packages/panels/src/Resources/Pages/ListRecords/Tab.php

```diff
@@ -15,6 +15,8 @@ class Tab extends Component
 
     protected string | Closure | null $badge = null;
 
+    protected string | Closure | null $badgeColor = null;
```

Comment:
Should accept `string | array | Closure | null`.
Could you update and mark as ready for review when done, @schmeits?

Reply:
Yes changed it... thanks for the tip.. completely forgot the array color option